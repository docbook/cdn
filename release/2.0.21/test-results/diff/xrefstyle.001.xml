<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Chapter 1. Unit Test: xrefstyle.001.xml</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="chapter" class="chapter">
         <header deltaxml:deltaV2="A=B" class="chapter-titlepage">
            <h2>Chapter 1. Unit Test: xrefstyle.001.xml</h2>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">Norman</span>
                     <span class="surname">Walsh</span>
                  </span>
               </h3>
            </div>
            <p class="CVS">$Id: xrefstyle.001.xml 4056 2004-11-23 13:28:50Z nwalsh $</p>
         </header>
         <p deltaxml:deltaV2="A">This is a test of xref styles.</p><div deltaxml:deltaV2="B" class="content">
            <p>This is a test of xref styles.</p>
            <p>Short self-reference: <a href="#chapter" title="Unit Test: xrefstyle.001.xml">Chapter 1, <i>Unit Test: xrefstyle.001.xml</i>
               </a>.</p>
            <p>Long self-reference: <a href="#chapter" title="Unit Test: xrefstyle.001.xml">Chapter 1, <i>Unit Test: xrefstyle.001.xml</i>
               </a>.</p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">Short self-reference: <a href="#chapter" title="Unit Test: xrefstyle.001.xml">Chapter 1, <i>Unit Test: xrefstyle.001.xml</i>
            </a>.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">Long self-reference: <a href="#chapter" title="Unit Test: xrefstyle.001.xml">Chapter 1, <i>Unit Test: xrefstyle.001.xml</i>
            </a>.</p>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>