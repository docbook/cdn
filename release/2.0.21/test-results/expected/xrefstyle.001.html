<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Chapter 1. Unit Test: xrefstyle.001.xml</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="chapter" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Unit Test: xrefstyle.001.xml</h2><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><p class="CVS">$Id: xrefstyle.001.xml 4056 2004-11-23 13:28:50Z nwalsh $</p></header><p>This is a test of xref styles.</p><p>Short self-reference: <a href="#chapter" title="Unit Test: xrefstyle.001.xml">Chapter 1, <i>Unit Test: xrefstyle.001.xml</i></a>.</p><p>Long self-reference: <a href="#chapter" title="Unit Test: xrefstyle.001.xml">Chapter 1, <i>Unit Test: xrefstyle.001.xml</i></a>.</p></article><script src="../../resources/js/prism.js"></script></body></html>